app/kumactl/cmd/install/install_control_plane_test.go

```diff
@@ -220,15 +220,7 @@ var _ = Describe("kumactl install control-plane", func() {
 			},
 			goldenFile: "install-control-plane.cni-enabled.golden.yaml",
 		}),
-		Entry("should generate Kubernetes resources with CNI plugin v1", testCase{
-			extraArgs: []string{
-				"--cni-enabled",
-				"--set",
-				"legacy.cni.enabled=true",
-			},
-			goldenFile: "install-control-plane.cni-legacy-enabled.golden.yaml",
-		}),
-		Entry("should generate Kubernetes resources with transparent v2 "+
+		Entry("should generate Kubernetes resources with ebpf"+
```

Comment:
nitpick: the name is a bit weird now `should generate Kubernetes resources with ebpf using ebpf (experimental)`